### ISSUE TITLE
static saliency model

### ISSUE THREAD

**Issue by hahafan** [shenjianbing/deepvideosaliency#1]
hi,

The static pre-trained saliency model is missing.

**Comment by JingangLang** [comment]
Hi，friend.Do you know how to run the codes only by cpu? Thanks.

**Comment by hahafan** [comment]
I think maybe you can change the opts.use_gpu to false.. and change the code in active_caffe_mex to use cpu.  I am not sure.

**Comment by JingangLang** [comment]
OK,thank you.I have installed the faster r-cnn wrote by python originally.It didn't work ! Finally I decide to install the matlab version after finding nothing to solve the problem.What's more,do you get the results?@hahafan

**Comment by hahafan** [comment]
No, I didn't get the result. because the pre-train static model is missing.

**Comment by JingangLang** [comment]
Oh, all right！I‘m trying to run it.Maybe we can  communicate with each other if I can get the result.

**Comment by JingangLang** [comment]
The same problem !It says that "staticsaliency_final does not exist".

**Comment by wenguanwang** [comment]
@hahafan @JingangLang Please download the model using the link of Baidupan. Thanks.
